### PR TITLE
Add Activity.SetParentId remarks

### PR DESCRIPTION
**Pull request by tarekgh**
## Summary

Add a remark for `Activity.SetParentId` talking about the expected behavior when calling it after starting the Activity.

https://github.com/dotnet/runtime/issues/60078

### CODE REVIEW

**Comment by tarekgh** [comment]
@noahfalk could you please have a look?

File: xml/System.Diagnostics/Activity.xml [hunk 1]
```diff
@@ -1155,7 +1156,8 @@ This property should only be used in 'boundary' scenarios where an <xref:System.
 
 ## Remarks
 
-This method has the advantage that no string manipulation is needed to set the ID.
+- This method has the advantage that no string manipulation is needed to set the ID.
+- This method should only be used before starting the <xref:System.Diagnostics.Activity> object. Calling this method after the starting of the <xref:System.Diagnostics.Activity> object will have no effect.
```
Comment (gewarren) [inline xml/System.Diagnostics/Activity.xml:1160]:
```suggestion
- This method should only be used before starting the <xref:System.Diagnostics.Activity> object. This method has no effect if you call it after the <xref:System.Diagnostics.Activity> object has started.
```

File: xml/System.Diagnostics/Activity.xml [hunk 2]
```diff
@@ -1115,7 +1115,8 @@ The specified format will take precedence over `Parent.IdFormat`, the <xref:Syst
 
 ## Remarks
 
-This property should only be used in 'boundary' scenarios where an <xref:System.Diagnostics.Activity> from another process logically started this <xref:System.Diagnostics.Activity>. The `parentId` brings up the <xref:System.Diagnostics.Activity.Tags> (as well as the <xref:System.Diagnostics.Activity.ParentId> property) and can be used to reconstruct the causal tree.
+- This method should only be used in 'boundary' scenarios where an <xref:System.Diagnostics.Activity> from another process logically started this <xref:System.Diagnostics.Activity>. The `parentId` brings up the <xref:System.Diagnostics.Activity.Tags> (as well as the <xref:System.Diagnostics.Activity.ParentId> property) and can be used to reconstruct the causal tree.
+- This method should only be used before starting the <xref:System.Diagnostics.Activity> object. Calling this method after the starting of the <xref:System.Diagnostics.Activity> object will have no effect.
```
Comment (gewarren) [inline xml/System.Diagnostics/Activity.xml:1119]:
```suggestion
- This method should only be used before starting the <xref:System.Diagnostics.Activity> object. This method has no effect if you call it after the <xref:System.Diagnostics.Activity> object has started.
```

**Review by gewarren — APPROVED** [review]

**Review by noahfalk — COMMENTED** [review]
LGTM : )

**Pull request merged**